fv3core/utils/null_comm.py

```diff
@@ -0,0 +1,53 @@
+class NullAsyncResult:
+    def __init__(self, recvbuf=None):
+        self._recvbuf = recvbuf
+
+    def wait(self):
+        if self._recvbuf is not None:
+            self._recvbuf[:] = 0.0
+
+
+class NullComm:
+    """
+    A class with a subset of the mpi4py Comm API, but which
+    'receives' zeros instead of using MPI.
```

Comment:
what is the reason to write `0` instead of `NaN`? Is this just to make sure that the computation does something realistic (even though not valid)?

Reply:
It's so it doesn't crash when we're compiling. If NaNs also wouldn't crash, that would be better. I don't know whether they do or don't but it's hard to test.

Reply:
I'll add a `fill_value=0.` argument.

Reply:
Added.